classes/class6/README.md

```diff
@@ -1 +1,96 @@
-#Class 6
+#Class 6: APIs and Debugging Strategies
+##APIs: Your *real* on-ramp to the information superhighway
+
+![Take it to the moon, take it to the stars.](/images/tothemoon.jpg)
+
+*Get in loser, we're going to programmatically acquire JSON data.*
```

Comment:
I'm open to other ideas here. Caption contest! Trying to go with the theme of [this meme](https://www.google.com/search?q=get+in+loser+we%27re+going+shopping&source=lnms&tbm=isch).